core/domain/fs_domain.py

```diff
@@ -296,12 +296,17 @@ def isfile(self, filepath):
         metadata = self._get_file_metadata(filepath, None)
         return bool(metadata)
 
-    def listdir(self, dir_name):
+    def listdir(self, dir_name, delimiter=None):  # pylint: disable=unused-argument
         """Lists all files in a directory.
 
+        If `delimiter` argument is unused. It is included so that this method
```

Comment:
Don't understand the first sentence. Is it unused or not?

Reply:
No it is not used, I figured out. It is not required. Removed.

Reply:
Delimiter removed entirely